### DISCUSSION TITLE
package io/fs is not in GOROOT

### DISCUSSION THREAD

**Discussion by SgtDaJim** [99designs/gqlgen#1970] [Q&A] [answered]
### What happened?
Run init command below 
```bash
go run github.com/99designs/gqlgen init
```
and returns:
```
../../../go/pkg/mod/github.com/99designs/gqlgen@v0.16.0/plugin/resolvergen/resolver.go:5:2: package io/fs is not in GOROOT (/usr/local/opt/go@1.15/libexec/src/io/fs)
```
So gqlgen v0.16.0 cannot be used under golang version 1.15？

### versions
 - `gqlgen version`? 
 v0.16.0
 - `go version`?
 go1.15.15
 - dep or go modules?
 go modules

**Comment by mpldr** [comment, marked as answer]
> So gqlgen v0.16.0 cannot be used under golang version 1.15?

`io/fs` was introduced in 1.16 ([source](https://godocs.io/io/fs?view=versions)), so no.
